src/main/java/org/ibp/api/rest/germplasm/GermplasmResource.java

```diff
@@ -190,6 +192,16 @@ public long getFilteredCount() {
 		return new ResponseEntity<>(this.germplasmService.importGermplasm(cropName, programUUID, germplasmList), HttpStatus.OK);
 	}
 
+	@ApiOperation(value = "Import germplasm updates. Updating Breeding Method is not yet supported.")
+	@PreAuthorize("hasAnyAuthority('ADMIN', 'CROP_MANAGEMENT', 'GERMPLASM', 'MANAGE_GERMPLASM', 'IMPORT_GERMPLASM_UPDATES')")
+	@RequestMapping(value = "/crops/{cropName}/germplasm", method = RequestMethod.PATCH)
+	@ResponseBody
+	public ResponseEntity<Set<Integer>> importGermplasmUpdates(@PathVariable final String cropName,
+		@RequestParam(required = false) final String programUUID,
+		@RequestBody final List<GermplasmUpdateDTO> germplasmList) {
+		return new ResponseEntity<>(this.germplasmService.importGermplasmUpdates(programUUID, germplasmList), HttpStatus.OK);
```

Comment:
No need to pass `programUUID`. You can use `ContextHolder.getCurrentProgram()` in `GermplasmUpdateValidator`